pages/docs/reference/js-modules.md

```diff
@@ -253,10 +250,53 @@ external fun sayHello(name: String)
 
 </div>
 
+### Automatic generation of external declarations with Dukat
```

Comment:
Why it's part of this file?

Reply:
Because this page deals with importing JsModules, package hierarchies & co – in short, a bunch of tasks that are required when working with Kotlin/JS externals.

Would you prefer the section on Dukat to be in a different file? If so, which? :)

Reply:
But Dukat works not only with JS module systems.
I think it could be a separate file.

Reply:
I've moved Dukat to a separate file, and added references in the interop parts of the documentation.